app/src/organisms/Desktop/Devices/RobotCard.tsx

```diff
@@ -163,6 +173,30 @@ function AttachedModules(props: { robotName: string }): JSX.Element | null {
   ) : null
 }
 
+function AttachedDevices(props: { robotName: string }): JSX.Element | null {
+  const { robotName } = props
+  const { t } = useTranslation('devices_landing')
+  const { data } = useCamera()
+  return data?.cameraEnabled ? (
+    <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing4}>
+      <LegacyStyledText
```

Comment:
```suggestion
      <StyledText
      desktopStyle="bodyDefaultRegular" (or something close to this)
```

In [figma](https://www.figma.com/design/PDkgB4AZ9PiLLJkwBATcU3/Feature--Camera-Enablement?node-id=5019-15985&t=PV72kzBTrynsLXTj-4), clicking into the "Input Devices" div will show a "Typography" section in the figma slideout, and there's a name property with a value of Helix/Product/Body-default-reg.

The presence of this name property indicates that we should use `StyledText` here, which contains the modern Helix design typography system (this text and the "Modules" text follow the old typography system). We can get rid of the `as` and `textTransform` properties when using `StyledText`, too.